Documentation/ceph-object.md

```diff
@@ -105,6 +105,46 @@ To directly retrieve the secrets:
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep AccessKey | awk '{print $2}' | base64 --decode
 kubectl -n rook-ceph get secret rook-ceph-object-user-my-store-my-user -o yaml | grep SecretKey | awk '{print $2}' | base64 --decode
 ```
+## Create an Object Bucket Claim
+
+Creating an Object Bucket Claim (OBC) causes the Rook-Ceph bucket provisioner to create a new bucket or to grant access to an existing bucket. When the OBC does not references an existing storage class set up by an administrator, a new bucket is created with the specified retention policy. See the [Object Bucket Claim Documentation](ceph-object-bucket-claim.md) for more detail on the `CephObjectBucketClaims`.
```

Comment:
Try rewording :

_Creating an Object Bucket Claim (OBC) causes the Rook-Ceph bucket provisioner to create a new bucket or to grant access to an existing bucket. OBCs reference a Storage Class which is created by a cluster administrator. The Storage Class defines the object storage system, the bucket retention policy, and, when granting access to existing (brownfield) buckets, the name of the bucket. If a new bucket is desired then the OBC contains the bucket name or a prefix to be used for a random bucket name. See the [Object Bucket Claim Documentation](ceph-object-bucket-claim.md) for more detail on the `CephObjectBucketClaims`._